src/xcode/ENA/ENA/Source/Scenes/Onboarding/DeltaOnboarding/NewVersionFeatures/DeltaOnboardingNewVersionFeaturesViewModel.swift

```diff
@@ -89,10 +89,18 @@ struct DeltaOnboardingNewVersionFeaturesViewModel {
 		]
 		
 		for feature in newVersionFeatures {
-			let featureBulletPoint = NSMutableAttributedString(string: feature.title + "\n\t", attributes: boldTextAttribute)
-			featureBulletPoint.append(NSAttributedString(string: feature.description, attributes: normalTextAttribute))
-			featureBulletPoint.append(NSAttributedString(string: "\n", attributes: normalTextAttribute))
-			cells.append(.bulletPoint(attributedText: featureBulletPoint))
```

Comment:
Let me know if you prefer to comment out the old lines 92-95 so that it's easier to add them again with version 2.8.